cmd/subrouter/sr.go

```
@@ -1005,6 +1142,65 @@ func (r srRunner) fetchUsageRows(ctx context.Context) ([]srUsageRow, error) {
+			email: "kimi-code", provider: accounts.ProviderKimi,
+			authMode: accounts.AuthModeOAuth, planType: "subscription", err: kimiErr,
+			score: selectacct.Score{AccountID: "kimi-code"},
+		})
+	}
+	for _, account := range kimiAccounts {
+		rowIndex := len(rows)
+		accountID := strings.TrimSpace(account.ID)
+		if accountID == "" {
+			accountID = "kimi-code"
+		}
+		rows = append(rows, srUsageRow{
```

golangci-lint (2.12.2)

[error] append to slice `rows` with non-zero initialized length

(makezero)